app/src/main/java/com/portfolio/tasky/entry/EntryActivity.kt

```diff
@@ -65,15 +82,20 @@ class EntryActivity : AppCompatActivity(), FragmentInflater by FragmentInflaterI
         }, 3000)
     }
 
-    private fun startFragment() {
+    fun startFragment(fragment : Fragment) {
+        setFragmentManager(supportFragmentManager)
         setTitle(getString(R.string.welcome_back))
 
-        setFragmentManager(supportFragmentManager)
-        val fragment = RegistrationFragment.getInstance()
         inflateFragment(fragment, R.id.fragment_container)
     }
 
     fun setTitle(text: String) {
         setToolBarText(viewBinding.toolbar, text)
     }
+
+    override fun onClick(view: View?) {
+        if(view == viewBinding.fab && viewBinding.fab.tag == "viewTag"){
```

Comment:
I'm not a fan of this
Maybe have a callback interface or something 

Or in the worst case, make it a constant

The reason why I don't like this, is that this activity needs to know about a string that's on a fragment